doc/code/qml_transforms_qcut.rst

```diff
@@ -0,0 +1,164 @@
+
+>>> graph = qml.transforms.qcut.tape_to_graph(tape)
+
+.. figure:: ../_static/qcut_graph.svg
+    :align: center
+    :width: 60%
+    :target: javascript:void(0);
+
+Our next step is to remove the :class:`~.WireCut` nodes in the graph and replace with
+:class:`~.MeasureNode` and :class:`~.PrepareNode` pairs.
+
+>>> qml.transforms.qcut.replace_wire_cut_nodes(graph)
+
+The :class:`~.MeasureNode` and :class:`~.PrepareNode` pairs are placeholder operations that
+allow us to cut the circuit graph and then iterate over measurement and preparation
+configurations at cut locations. First, the :func:`~.fragment_graph` function pulls apart
+the graph into disconnected components as well as returning the ``communication_graph``
+detailing the connectivity between the components.
+
+>>> fragments, communication_graph = qml.transforms.qcut.fragment_graph(graph)
+
+We now convert the ``fragments`` back to :class:`~.QuantumTape` objects
+
+>>> fragment_tapes = [qml.transforms.qcut.graph_to_tape(f) for f in fragments]
+
+The circuit fragments can now be visualized:
+
+>>> print(fragment_tapes[0].draw())
+ 0: ──RX(0.531)──╭C──RY(-0.4)─────┤ ⟨Z⟩
+ 1: ──RY(0.9)────╰Z──MeasureNode──┤
+
+>>> print(fragment_tapes[1].draw())
+ 2: ──RX(0.3)──────╭Z──╭┤ ⟨Z ⊗ Z⟩
+ 1: ──PrepareNode──╰C──╰┤ ⟨Z ⊗ Z⟩
+
+Additionally, we must remap the tape wires to match those available on our device.
+
+>>> dev = qml.device("default.qubit", wires=2)
+>>> fragment_tapes = [
+...     qml.transforms.qcut.remap_tape_wires(t, dev.wires) for t in fragment_tapes
+... ]
+
+Next, each circuit fragment is expanded over :class:`~.MeasureNode` and
+:class:`~.PrepareNode` configurations and a flat list of tapes is created:
+
+.. code-block::
+
+    expanded = [qml.transforms.qcut.expand_fragment_tape(t) for t in fragment_tapes]
+
+    configurations = []
+    prepare_nodes = []
+    measure_nodes = []
+    for tapes, p, m in expanded:
+        configurations.append(tapes)
+        prepare_nodes.append(p)
+        measure_nodes.append(m)
+
+    tapes = tuple(tape for c in configurations for tape in c)
+
+Each configuration is drawn below:
+
+>>> for t in tapes:
+...     print(t.draw())
+
+.. code-block::
+
+     0: ──RX(0.531)──╭C──RY(-0.4)──╭┤ ⟨Z ⊗ I⟩ ╭┤ ⟨Z ⊗ Z⟩
+     1: ──RY(0.9)────╰Z────────────╰┤ ⟨Z ⊗ I⟩ ╰┤ ⟨Z ⊗ Z⟩
+
+     0: ──RX(0.531)──╭C──RY(-0.4)──╭┤ ⟨Z ⊗ X⟩
+     1: ──RY(0.9)────╰Z────────────╰┤ ⟨Z ⊗ X⟩
+
+     0: ──RX(0.531)──╭C──RY(-0.4)──╭┤ ⟨Z ⊗ Y⟩
+     1: ──RY(0.9)────╰Z────────────╰┤ ⟨Z ⊗ Y⟩
+
+     0: ──RX(0.3)──╭Z──╭┤ ⟨Z ⊗ Z⟩
+     1: ──I────────╰C──╰┤ ⟨Z ⊗ Z⟩
+
+     0: ──RX(0.3)──╭Z──╭┤ ⟨Z ⊗ Z⟩
+     1: ──X────────╰C──╰┤ ⟨Z ⊗ Z⟩
+
+     0: ──RX(0.3)──╭Z──╭┤ ⟨Z ⊗ Z⟩
+     1: ──H────────╰C──╰┤ ⟨Z ⊗ Z⟩
+
+     0: ──RX(0.3)─────╭Z──╭┤ ⟨Z ⊗ Z⟩
+     1: ──H────────S──╰C──╰┤ ⟨Z ⊗ Z⟩
+
+The last step is to execute the tapes and postprocess the results using
+:func:`~.qcut_processing_fn`, which converts the results into a tensor network contraction.
```

Comment:
Wording is a bit strange (at least IMO).  Perhaps use:
"which processes the results to the original full circuit output vie a tensor network contraction"

Reply:
Thanks, done!